tuneme/static/new/js/main.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
```

Comment:
Is the JS necessary though?

Reply:
@moh-moola we need to get the form values and populate them as they are being selected. Might not need JS once form is solved

Reply:
Doesn't the selected attribute of the option element take care of that, we can add it dynamically for a prepopulated form

Reply:
oh I get it now ...for the label, but will we just disregard this feature for feature phones or maybe we need to rethink this fields label?